dev-docs/source/PatchReleaseChecklist.rst

```diff
@@ -141,11 +140,10 @@ Once the testing has passed:
   :ref:`release checklist <ReleaseChecklist>`.
 * Send an email, including the text of the release notes, to the
   following lists
-* ``mantid-announce@mantidproject.org``
-* ``mantid-developers@mantidproject.org``
 * ``nobugs@nobugsconference.org``
 * ``news@neutronsources.org``
 * ``neutron@neutronsources.org``
+* Also post the contents of the message on General on Slack
```

Comment:
I think we have been using #Announcements for this?

Reply:
Good point I've changed this